packages/components/src/internal/components/buttons/AddEntityButton.spec.tsx

```diff
@@ -31,14 +30,19 @@ describe('<AddEntityButton />', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
-    test('Typical properties', () => {
+    test('Fully populated props', () => {
```

Comment:
Same goes for these snapshot test name changes, please check `AddEntityButton.spec.tsx.snap ` for sections that should now be removed.

Reply:
Done!